panel/widgets/slider.py

```diff
@@ -452,6 +454,9 @@ class DateRangeSlider(_RangeSliderBase):
     end = param.Date(default=None)
 
     step = param.Number(default=1)
+    
+    format = param.String(r"%m%Y",
```

Comment:
Is this the right default? Shouldn't it include the day?

Reply:
You are right. The default should be 

r"%d %b %Y"

to get the same formatting as with the original DateRangeSlider.